AppCenter/AppCenter/Internals/Util/MSOrderedDictionary.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+@interface MSOrderedDictionary : NSMutableDictionary
+
+- (instancetype)initWithCapacity:(NSUInteger)numItems;
+
+- (void)setObject:(id)anObject forKey:(id<NSCopying>)aKey;
+
+- (BOOL)isEqualToDictionary:(NSDictionary *)otherDictionary;
```

Comment:
We don't need this one, it is override.

Reply:
We have to because we are using the decorator pattern (can't call `[super initWithCapacity:]`).